docs/cn/core-services/Caching.md

```diff
@@ -1,259 +1,507 @@
+alluxio.worker.tieredstore.level1.alias=HDD
+# configured 3 separate file paths for the second tier
+alluxio.worker.tieredstore.level1.dirs.path=/mnt/hdd1,/mnt/hdd2,/mnt/hdd3
+# defined HDD to be the medium type of the second tier
+alluxio.worker.tieredstore.level1.dirs.mediumtype=HDD,HDD,HDD
+# define the quota for each of the 3 file paths of the second tier
+alluxio.worker.tieredstore.level1.dirs.quota=2TB,5TB,500GB
 ```
 
-与Alluxio2.0之前采用的同步回收相比，在写或读缓存高工作负载时异步回收可以提高性能。
-同步回收等待一个客户端请求比当前在worker上可用空间更多的空间，然后启动回收进程来释放足够的空间来满足这一要求。这导致了许多很小的回收尝试，使得效率较低但是使可用的Alluxio空间的利用最大化。
+可配置层数没有限制
+但是每个层都必须使用唯一的别名进行标识。
+典型的配置将具有三层，分别是内存，SSD和HDD。
+要在HDD层中使用多个硬盘存储，需要在配置`alluxio.worker.tieredstore.level {x} .dirs.path`时指定多个路径。
 
-### 回收策略
+###块注释策略
 
-Alluxio使用回收策略决定当空间需要释放时，哪些数据块被移到低存储层。用户可以指定Alluixo回收策略来获得细粒度地控制回收进程。
+Alluxio从v2.3开始使用块注释策略来维护存储中数据块的严格顺序。 
+注释策略定义了跨层块的顺序，并在以下操作过程中进行用来参考:
+-释放空间
+-[动态块放置](#block-aligning-dynamic-block-placement)。
 
-Alluxio支持自定义回收策略，已有的实现包括：
+与写操作同步发生的释放空间操作将尝试根据块注释策略强制顺序删除块并释放其空间给写操作。注释顺序的最后一个块是第一个释放空间候选对象，无论它位于哪个层上。
 
-* **贪心回收策略(GreedyEvictor)**
+开箱即用的注释策略实施包括:
 
-    回收任意的块直到释放出所需大小的空间。
+- **LRUAnnotator**:根据最近最少使用的顺序对块进行注释和释放。
+**这是Alluxio的默认注释策略**。
+- **LRFUAnnotator**:根据配置权重的最近最少使用和最不频繁使用的顺序对块进行注释。
+    - 如果权重完全偏设为最近最少使用，则行为将
+    与LRUAnnotator相同。
+    - 适用的配置属性包括`alluxio.worker.block.annotator.lrfu.step.factor`
+    `alluxio.worker.block.annotator.lrfu.attenuation.factor`。
 
-* **LRU回收策略(LRUEvictor)**
+workers选择使用的注释策略由Alluxio属性
+[`alluxio.worker.block.annotator.class`]({{ '/en/reference/Properties-List.html' | relativize_url}}#alluxio.worker.block.annotator.class)。
+该属性应在配置中指定完全验证的策略名称。当前
+可用的选项有:
 
-    回收最近最少使用的数据块直到释放出所需大小的空间。
+-`alluxio.worker.block.annotator.LRUAnnotator`
+-`alluxio.worker.block.annotator.LRFUAnnotator`
 
-* **LRFU回收策略(LRFUEvictor)**
+####释放空间模拟
+旧的释放空间策略和Alluxio提供的实施现在已去掉了，并用适当的注释策略替换。
+配置旧的Alluxio释放空间策略将导致worker启动失败，并报错`java.lang.ClassNotFoundException`。
+同样，旧的基于水位标记配置已失效。因此，以下配置选项是无效的:
+-`alluxio.worker.tieredstore.levelX.watermark.low.ratio`
+-`alluxio.worker.tieredstore.levelX.watermark.high.ratio`
 
-    基于权重分配的最近最少使用和最不经常使用策略回收数据块。如果权重完全偏向最近最少使用,LRFU回收策略退化为LRU回收策略。
+然而，Alluxio支持基于自定义释放空间实施算法数据块注释的仿真模式。该仿真模式假定已配置的释放空间策略创建一个基于某种顺序释放空间的计划，并通过定期提取这种自定义顺序来支持块注释活动。
 
-* **部分LRU回收策略(PartialLRUEvictor)**
+旧的释放空间配置应进行如下更改。 (由于旧的释放空间实施已删除，如未能更改基于旧实施的以下配置就会导致class加载错误。)
 
-    基于最近最少使用回收，但是选择有最大剩余空间的存储目录(StorageDir)，只从该目录回收数据块。
+-LRUEvictor-> LRUAnnotator
+-GreedyEvictor-> LRUAnnotator
+-PartialLRUEvictor-> LRUAnnotator
+-LRFUEvictor-> LRFUAnnotator
 
-将来会有更多的回收策略可供选择。由于Alluxio支持自定义回收策略。你也可以为自己的应用开发合适的回收策略。
+###分层存储管理
+因为块分配/释放不再强制新的写入必须写到特定存储层，新数据块可能最终被写到任何已配置的存储层中。这样允许写入超过Alluxio存储容量的数据。但是，这就需要Alluxio动态管理块放置。
+为了确保层配置为从最快到最慢的假设，Alluxio会基于块注释策略在各层存储之间移动数据块。
 
-使用同步回收时，推荐使用较小的块大小配置（64-128MB左右），以降低块回收的延迟。使用空间预留器时，块大小不会影响回收延迟。
+每个单独层管理任务都遵循以下配置:
+-`alluxio.worker.management.task.thread.count`:管理任务所用线程数。 (默认值:CPU核数)
+-alluxio.worker.management.block.transfer.concurrency.limit`:可以同时执行多少个块传输。 (默认:`CPU核数`/2)
```

Comment:
remember to add backtick before each property key

Reply:
backticks should be used as pairs, otherwise the markdown file format will be chaotic